packages/react-native-editor/__device-tests__/gutenberg-editor-gallery.test.js

```diff
@@ -6,6 +6,9 @@ import { blockNames } from './pages/editor-page';
 describe( 'Gutenberg Editor Gallery Block tests', () => {
 	it( 'should be able to add a gallery block', async () => {
 		await editorPage.addNewBlock( blockNames.gallery );
+		await editorPage.driver.sleep( 1000 );
```

Comment:
This delay and the one below ensures the picker has been opened before attempting to close it.